nsexec/src/nsexec.h

```diff
@@ -0,0 +1,33 @@
+#ifndef NSEXEC_H
+#define NSEXEC_H
+
+#include <setjmp.h>
+
+/* Assume the stack grows down, so arguments should be above it. */
+struct clone_t {
+	/*
```

Comment:
About indent spaces, you can see [this](https://google.github.io/styleguide/cppguide.html#Spaces_vs._Tabs). It is also the same case with other files.